src/test/pom-parser/pom-parser.test.js

```diff
@@ -107,111 +104,148 @@ describe('POMParser', () => {
-            assert.equal(repository.url, undefined);
+          repository = pomContent.repositories[2];
+          assert.equal(repository.id, 'Target');
+          assert.equal(repository.url, undefined);
         });
       });
 
-      it('Repo name/layout', () => { 
+      it('Repo name/layout', () => {
         const rawContent = readFile('repo-name-layout.xml');
 
-        return pomParser.parsePOMFromString(rawContent)
-          .then((pomContent) => {
-            repository = pomContent.repositories[0];
-            assert.equal(repository.name, 'Elizabeth Bennet');
-            assert.equal(repository.layout, 'Regency');
+        return pomParser.parsePOMFromString(rawContent).then(pomContent => {
+          repository = pomContent.repositories[0];
+          assert.equal(repository.name, 'Elizabeth Bennet');
+          assert.equal(repository.layout, 'Regency');
 
-            repository = pomContent.repositories[1];
-            assert.equal(repository.name, 'Mr. Darcy');
-            assert.equal(repository.layout, undefined);
+          repository = pomContent.repositories[1];
+          assert.equal(repository.name, 'Mr. Darcy');
+          assert.equal(repository.layout, undefined);
 
-            repository = pomContent.repositories[2];
-            assert.equal(repository.name, undefined);
-            assert.equal(repository.layout, 'Jane Austen');
+          repository = pomContent.repositories[2];
+          assert.equal(repository.name, undefined);
+          assert.equal(repository.layout, 'Jane Austen');
         });
       });
 
-      it('Repo releases/snapshots', () => { 
+      it('Repo releases/snapshots', () => {
         const rawContent = readFile('repo-releases-snapshots.xml');
-  
-        return pomParser.parsePOMFromString(rawContent)
-          .then((pomContent) => {
-            repository = pomContent.repositories[0];
-            assert.equal(repository.releases.enabled, 'up');
-            assert.equal(repository.releases.updatePolicy, 'Never');
-            assert.equal(repository.releases.checksumPolicy, 'gunna');
-            assert.equal(repository.snapshots.enabled, 'let');
-            assert.equal(repository.snapshots.updatePolicy, 'you');
-            assert.equal(repository.snapshots.checksumPolicy, 'down');
-  
-            emptyRSrepository = pomContent.repositories[1];
-            assert.equal(emptyRSrepository.releases.enabled, undefined);
-            assert.equal(emptyRSrepository.releases.updatePolicy, undefined);
-            assert.equal(emptyRSrepository.releases.checksumPolicy, undefined);
-            assert.equal(emptyRSrepository.snapshots.enabled, undefined);
-            assert.equal(emptyRSrepository.snapshots.updatePolicy, undefined);
-            assert.equal(emptyRSrepository.snapshots.checksumPolicy, undefined);
+
+        return pomParser.parsePOMFromString(rawContent).then(pomContent => {
+          repository = pomContent.repositories[0];
```

Comment:
This is happening in a few places.  You need to declare the variable at the same time or before you assign it.   (it is missing the const keyword at the beginning of this line).

There are a few places in the test where this is happening.